<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfigurePreset</class>
 <widget class="QDialog" name="DialogConfigurePreset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Font</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_currentPresetName">
     <property name="text">
      <string>Font Preset Name:</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_currentFontPresetName</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_currentFontPresetName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Default preset:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>font:</string>
       </property>
       <property name="buddy">
        <cstring>comboBox_defaultFont</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_defaultFont"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>font size:</string>
       </property>
       <property name="buddy">
        <cstring>spinBox_defaultFontSize</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_defaultFontSize">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_applyDefaultToAllLocales">
     <property name="text">
      <string>apply default preset for all locales</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>preset for locale:</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_locale</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_locale"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_resetLocale">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>font:</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_localizedFont</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_localizedFont"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>font size:</string>
         </property>
         <property name="buddy">
          <cstring>spinBox_localizedFontSize</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_localizedFontSize">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1" rowspan="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
